API/Entities/UserSettings.cs

```diff
@@ -0,0 +1,50 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using API.Entities.Interfaces;
+using API.Osu.Enums;
+
+namespace API.Entities;
+
+/// <summary>
+/// Represents a user's settings for otr-web
+/// </summary>
+[Table("user_settings")]
+public class UserSettings : IUpdateableEntity
+{
+    [Key]
+    [Column("id")]
+    public int Id { get; set; }
+
+    [Column("created")]
+    public DateTime Created { get; set; }
+
+    [Column("updated")]
+    public DateTime? Updated { get; set; }
+
+    /// <summary>
+    /// The id of the associated user
+    /// </summary>
+    [Column("user_id")]
+    public int UserId { get; set; }
+
+    /// <summary>
+    /// The default ruleset for the associated user
+    /// </summary>
+    [Column("default_ruleset")]
+    public Ruleset? DefaultRuleset { get; set; }
```

Comment:
nitpick, but I prefer the Created/Updated props to be on the bottom (I would even undo and recreate the migration):

From top to bottom:
- Id
- Properties
- Created
- Updated
- Navigation props / collections